Alert.css

```diff
@@ -0,0 +1,77 @@
+:root {
+  --color: black;
+}
+
+.header {
+  text-align: center;
+  color: indianred;
+}
+
+.container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  margin-left: 10%;
+  margin-right: 10%;
+}
+
+.alert {
+  border: 2px solid var(--color);
+  height: 20px;
+  padding: 0.9375rem 0.625rem;
+  width: 100%;
+  border-radius: 4px;
+  margin-bottom: 10px;
+  /* width: 50%; */
+}
+
+.alert-danger {
+  color: white;
+  background-color: #fecaca;
+  border-color: #f5cece;
+}
+
+.alert-primary {
+  color: #084298;
+  background-color: #cfe2ff;
+  border-color: #cedef5;
+}
+
+.alert-secondary {
+  color: #41464b;
+  background-color: #e2e3e5;
+  border-color: #e2e3e5;
+}
+
+.alert-success {
+  color: #059669;
+  background-color: #a7f3d0;
+  border-color: #a7f3d0;
+}
+
+.alert-Warning {
+  color: #fcd34d;
```

Comment:
I suggest you to be consistent with the colors and use variables to define color. Also, this will help in your projects too.

Reply:
Yes, I have Converted repeatedly used colors into variables and calling in css